### PR TITLE
added a event

### PR DESCRIPTION
**Pull request by Ankk98**
added pycon 2k17

### CODE REVIEW

**Comment by Ankk98** [comment]
added pr for issue #7

File: static/data/2017/9/events.json [hunk 1]
```diff
@@ -53,4 +53,13 @@
     "website"  : "http://reactalicante.es/",
     "logo"     : "https://pbs.twimg.com/profile_images/906151350588887040/XC5RD0t-_400x400.jpg"
   }
+  {
+    "name" : "PyCon",
+    "start": "04/09/2017",
+    "end"  : "05/09/2017",
+    "location" : "Delhi, India",
+    "type" : "confrence",
```
Comment (geekyd) [inline static/data/2017/9/events.json]: @Ankk98 Seems like you made a typo here.

File: static/data/2017/9/events.json [hunk 2]
```diff
@@ -53,4 +53,13 @@
     "website"  : "http://reactalicante.es/",
     "logo"     : "https://pbs.twimg.com/profile_images/906151350588887040/XC5RD0t-_400x400.jpg"
   }
```
Comment (geekyd) [inline static/data/2017/9/events.json]: add `,`

Comment (Ankk98) [reply static/data/2017/9/events.json]: done

File: static/data/2017/9/events.json (hunk 1 above)
Comment (Ankk98) [reply static/data/2017/9/events.json]: resolved

File: static/data/2017/9/events.json [hunk 3]
```diff
@@ -52,5 +52,14 @@
     "type"     : "conference",
     "website"  : "http://reactalicante.es/",
     "logo"     : "https://pbs.twimg.com/profile_images/906151350588887040/XC5RD0t-_400x400.jpg"
-  }
+  },
+  {
+    "name" : "PyCon",
+    "start": "04/09/2017",
```
Comment (geekyd) [inline static/data/2017/9/events.json]: The dates are incorrect. The dates are 4-5 november

**Pull request closed without merging**